index.html

```diff
@@ -14,13 +14,20 @@
 				margin-top: 3%;
 				margin-left: 3%;
 				margin-right: 3%;
+			  background-color: #fff;     /* background */
+			  color: #000;                /* for text */
+			}

```

Comment:
You should indent this 2 more spaces.

(also, seems like you have a mix of tabs and spaces in this file)